src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultTagHelperCompletionService.cs

```diff
@@ -0,0 +1,299 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Microsoft.VisualStudio.Editor.Razor;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using RazorTagHelperCompletionService = Microsoft.VisualStudio.Editor.Razor.TagHelperCompletionService;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    internal class DefaultTagHelperCompletionService : TagHelperCompletionService
+    {
+        private static readonly Container<string> ElementCommitCharacters = new Container<string>(" ");
```

Comment:
What is a commit character?

Reply:
When the completion is selected it's the character you can press to insert the completion. I.e. for C# typing `@Dat` and then pressing `.` results in `@DateTime.`

Reply:
I see. What about `Enter` and `Tab`?

Reply:
Those are defaults 😄

Reply:
Should add `>` to the list